### PR TITLE
Modify editNote to no longer override userIds

### PR DESCRIPTION
**Pull request by fabubaker**
Previously, `editNote` would override the userId of the existing note object with the userId passed to the function.

This PR changes the behavior so that the `userId` of the note object is left alone but the userId passed to the function is recorded in the customizations log.

### CODE REVIEW

File: server/src/adapters/postgres/postgres.ts [hunk 1]
```diff
@@ -1412,15 +1412,23 @@ module.exports = {
...
    * @param {String} noteText - The note text provided by frontend.
    */
-  editNote: function (reviewId, articleId, userId, noteId, noteText) {
+  editNote: async function (reviewId, articleId, userId, noteId, noteText) {
     const logKey = generateNoteLogKey(noteId)
     const logValue = notesLogValues.EDITED
 
-    const noteObj = { user_id: parseInt(userId), text: noteText }
+    // Grab the note object to make changes to it.
+    const noteQuery = await pool.query(`
```
Comment (hammady) [inline server/src/adapters/postgres/postgres.ts:1424]: Please note that this is not atomic. Do you want to make it in a single transaction?

Comment (fabubaker) [reply server/src/adapters/postgres/postgres.ts:1424]: Yes, good catch, I'll change it.

Comment (fabubaker) [reply server/src/adapters/postgres/postgres.ts]: Got rid of that read, it wasn't needed anyways.

File: server/src/adapters/postgres/postgres.ts [hunk 2]
```diff
@@ -1412,28 +1412,27 @@ module.exports = {
    * Edits existing note for article.
    * @param {UInt} reviewId - The id of the review to which the note belongs.
    * @param {UInt} articleId - The id of the article to which the note belongs.
-   * @param {UInt} userId - The id of the user whose note is to be edited.
+   * @param {UInt} userId - The id of the user who is making the edit.
    * @param {UInt} noteId - The id of the note to be edited.
    * @param {String} noteText - The note text provided by frontend.
    */
-  editNote: function (reviewId, articleId, userId, noteId, noteText) {
+  editNote: async function (reviewId, articleId, userId, noteId, noteText) {
```
Comment (hammady) [inline server/src/adapters/postgres/postgres.ts:1419]: Did you forget to remove this change?

**Review by hammady — CHANGES_REQUESTED** [review]

**Review by hammady — APPROVED** [review]

**Pull request merged**